teekize/Questionner
# #163047692 Adds the folder structure for the application

## What does this PR do?
Adds the folder structure for the app

## Description of task to be completed
create the folder structure with the required folders and their required files

## How to manually test this
1. clone this repository
`https://github.com/teekize/Questionner`
2. cd into the project directory
`cd Questionner`
3. git checkout to this branch
`git checkout ch-folder-structure-163047692`
4. check the folder structure

## Screenshot
![folder](https://user-images.githubusercontent.com/28742266/50820343-67fbd180-133d-11e9-84c1-ef219ca4f6b7.JPG)
